setup.cfg

```
@@ -21,6 +21,7 @@ project_urls =
 [options]
 python_requires = >=3.7
 install_requires =
+    h5py ==2.5.0
```

```suggestion
    h5py ~=2.5.0
```
Perhaps allow bug fixes to be accepted without changes?